static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -103,34 +101,122 @@ $padding-wide: 100px;
 }
 
 .recipe__label {
+    @include f-headline;
     position: absolute;
     z-index: 300;
-    background-color: $purple;
-    height: 90px;
-    width: 160px;
+    background-color: $features-main-2;
+    height: 65px;
+    width: 110px;
     border-radius: 50%;
     text-align: center;
-    padding-top: 70px;
+    padding-top: 45px;
     color: #ffffff;
-    left: 1px;
-    top: 0;
+    left: -$gs-gutter / 6;
+    bottom: -$gs-gutter * 1.6;
     line-height: initial;
+    font-weight: 900;
 
-    @include mq($from: tablet) {
-        top: 5px;
-        left: 20px;
+    @include mq($from: desktop) {
+        top: -$gs-gutter * 2;
+        left: 14%;
+        bottom: auto;
     }
 }
 
 .recipe__article {
-    padding: 0 12px;
-    padding-bottom: 44px;
+    padding: 0 $gs-gutter / 2;
+    padding-bottom: $gs-gutter * 4;
+}
+
+.recipe__gutter-wrapper {
+    display: none;
+
+    @include mq($from: desktop) {
+        display: flex;
+        flex-basis: 5%;
+        position: sticky;
+        top: 0;
+        height: 100vh;
+        border-right: 1px solid $grey;
+    }
+}
+
+.recipe__article--byline-image {
+    position: absolute;
+    border-radius: 50%;
+    width: calc(#{$gs-gutter / 2} * 5);
+    height: calc(#{$gs-gutter / 2} * 5);
+    background-color: $grey;
+    overflow: hidden;
+    top: $gs-baseline;
+    margin: auto;
+    left: 0;
+    right: 0;
+}
+
+.recipe__article--byline-image .byline-img .byline-img__img {
+    width: 150%;
```

Comment:
@NataliaLKB We do 110% else where for the features template - I just wanted a slightly closer crop on the images as the byline image is quite small. 

https://github.com/guardian/frontend/blob/43bcdf3cd10d5a0839d843cd6a23042d1312e10d/static/src/stylesheets/module/content/_content.scss#L484